index.html

```diff
@@ -155,14 +155,19 @@ <h3>{{RTCPeerConnection/addTransceiver()}}</h3>
            {{RTCRtpTransceiverInit/sendEncodings}} within
            {{RTCPeerConnection/addTransceiver()}}. To validate
            {{RTCRtpEncodingParameters/scalabilityMode}}, add
-           the following validation steps after step 3 (within step 8):
+           the following steps after step 8 of
+           [=RTCPeerConnection/addTransceiver sendEncodings validation steps=]:
            <ol>
              <li>
-               If <var>sendEncodings</var> contains any encoding whose
+               If <var>sendEncodings</var> contains any encoding with a {{RTCRtpEncodingParameters}}.codec
```

Comment:
for `.codec` can we reference webrtc-extensions?

Reply:
Not until the other change has been merged. Will update this whenever we merge the other PR.

Reply:
Actually, I can add a reference to `[[!WEBRTC-EXTENSIONS]]` for now.